oeps/architectural-decisions/oep-0052/decisions/001-schema-representation.rst

```diff
@@ -0,0 +1,30 @@
+Schema Representation
+==================================================================================
+
+Context
+-------
+
+The event bus client libraries have their own object representation of messages. In pulsar, they are defined as `Record`
+objects. These objects map easily to Avro Schema definitions and can be used to define the schema. Also, Avro has been the default Kafka serialisation mechanism for a while.
```

Comment:
I think the part in @robrap 's suggestion about Avro being the default serialization format for Kafka is not actually true anymore, although it used to be. However, Avro is still recommended by the creators of Kafka since it is particularly efficient space-wise, and there's a lot of OOTB support for it. (https://codingharbour.com/apache-kafka/why-use-avro-data-format-with-apache-kafka/)

Reply:
I mostly took roberts suggestion, but changed `Avro is the default serialization format` to `Avro is the recommended serialization format`